app/views/setting.html

```diff
@@ -14,31 +9,31 @@
           <label for="condensedItems" title="Items that do not match filter criteria will be hidden.">Hide Unfiltered Items while Filtering</label>
         </td>
         <td>
-          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save('hideFilteredItems')" />
+          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save()" />
```

Comment:
Yeah I thought about just using a watch to call save. Maybe I'll just do
that!

On Monday, June 27, 2016, kyle shay <notifications@github.com> wrote:

> In app/views/setting.html
> <https://github.com/DestinyItemManager/DIM/pull/637#discussion_r68589758>:
>
> > @@ -14,31 +9,31 @@
> >            <label for="condensedItems" title="Items that do not match filter criteria will be hidden.">Hide Unfiltered Items while Filtering</label>
> >          </td>
> >          <td>
> > -          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save('hideFilteredItems')" />
> > +          <input type="checkbox" ng-model="vm.settings.hideFilteredItems" ng-change="vm.save()" />
>
> not a big thing, but in this case would it make sense to just create a
> watcher in the controller for vm.settings and if it changes, call
> vm.save()? or is it more obvious that a save is happening to show it here
> on the ng-change event?
>
> either way, love this PR. dimSettingsService.factory.js looks great now,
> utilizing filters and such seems to be much cleaner and read better.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DestinyItemManager/DIM/pull/637/files/ba4125f4c8e20296dd58985df9edc8bb13e4c15e#r68589758>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATHeMcWUR_W-qR7UHxlQAbE9zqKvS-tks5qP-LpgaJpZM4I-dD0>
> .
>